models.py

```diff
@@ -146,3 +251,36 @@ def callbacks_unet(callbacks_config):
         callbacks=[experiment_timing, training_monitor, validation_monitor,
                    model_checkpoints, lr_scheduler, early_stopping, neptune_monitor,
                    ])
+
+
+def multiclass_weighted_segmentation_loss(output, target, w0, sigma):
+    '''
+    w1 is temporarily torch.ones - it should handle class imbalance for thw hole dataset
+    '''
+    distance = target[:, 1, :, :]
+    target = target[:, 0, :, :].long()
+    w1 = Variable(torch.ones(distance.size()), requires_grad=False)  # TODO: fix it to handle class imbalance
```

Comment:
@apyskir add this as issue so we don't forget

Reply:
@jakubczakon Done